File: push/src/main/java/com/xiaomi/common/logger/thrift/mfs/HostInfo.java
package com.xiaomi.common.logger.thrift.mfs;

import java.util.List;

public class HostInfo {
    private String host;
    private List<LandNodeInfo> landNodeInfo;

    public void setHost(String host) {
        this.host = host;
    }

    public void setLand_node_info(List<LandNodeInfo> landNodeInfo) {
        this.landNodeInfo = landNodeInfo;
    }
}

File: legacy-runtime/src/main/java/com/xiaomi/common/logger/thrift/mfs/HostInfo.kt
package com.xiaomi.common.logger.thrift.mfs

class HostInfo {
    private var host: String? = null
    private var landNodeInfo: List<LandNodeInfo>? = null

    fun setHost(host: String?) {
        this.host = host
    }

    fun setLand_node_info(landNodeInfo: List<LandNodeInfo>?) {
        this.landNodeInfo = landNodeInfo
    }
}
